cms/plugin_rendering.py

```diff
@@ -187,8 +191,9 @@ def render_placeholder(placeholder, context_to_copy, name_fallback="Placeholder"
     context['edit'] = edit
     result = render_to_string("cms/toolbar/content.html", flatten_context(context))
     changes = watcher.get_changes()
-    if placeholder and not edit and placeholder.cache_placeholder and get_cms_setting('PLACEHOLDER_CACHE') and use_cache:
-        set_placeholder_cache(placeholder, lang, content={'content': result, 'sekizai': changes}, request=request)
+    if use_cache and not edit and placeholder.cache_placeholder and get_cms_setting('PLACEHOLDER_CACHE'):
```

Comment:
``use_cache`` will only be true if the user is not logged in, this means that is not necessary to check ``edit`` here